app/models/concerns/orchestration/dns.rb

```diff
@@ -18,6 +19,20 @@ def reverse_dns?
     (host.nil? || host.managed?) && managed? && hostname.present? && ip_available? && !subnet.nil? && subnet.dns? && SETTINGS[:unattended]
   end
 
+  def rebuild_dns
+    begin
+      if dns? && reverse_dns?
```

Comment:
It's quite possible to have one and not the other, these should both be run, not only when they are both true.